iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/CoordinatedReactiveControlXmlSerializer.java

```diff
@@ -48,23 +34,8 @@ public void write(CoordinatedReactiveControl extension, XmlWriterContext context
     }
 
     @Override
-    public CoordinatedReactiveControl read(Generator extendable, XmlReaderContext context) throws XMLStreamException {
```

Comment:
why do you drop the XMLStreamException?

Reply:
It is never thrown during this particular implementation, I suspect it was in the signature by default by the IDE template but it is not necessary here.